agents/LICENSE

```diff
@@ -0,0 +1,73 @@
+Apache License
```

Comment:
Question - is having a different license for `agents` intentional? I'm *guessing* not, and it's a carry-over from the migration, but if it is different, just wanted to clarify what that means for us :laughing:

Reply:
Correct, not intentional. I'll remove, thanks.